cmd/celestia/util/rpc.go

```diff
@@ -0,0 +1,58 @@
+package util
```

Comment:
This may also live in just `cmd`:
* It can be used outside of `celestia` binary, by something like `cel-shed`
* `cmd` contains reusable cmds and may as well contain reusable utils across out binaries